nbbuild/cluster.properties

```diff
@@ -572,6 +572,8 @@ nb.cluster.java=\
         debugger.jpda.kit,\
         debugger.jpda.projects,\
         debugger.jpda.projectsui,\
+        debugger.jpda.truffle,\
+        debugger.jpda.trufflenode,\
```

Comment:
Great. As the future of Nashorn is to diminish in favor of Graal.js & Truffle, I believe we want to have these modules in the standard distribution. Btw. this is related to [NETBEANS-1009](https://issues.apache.org/jira/browse/NETBEANS-1009).